scuttlebutt/src/server.rs

```diff
@@ -409,4 +411,29 @@ mod tests {
 
         server.shutdown().await.unwrap();
     }
+
+    #[tokio::test]
+    async fn test_member_change_event_is_broadcasted() {
+        let node1 = ScuttleServer::spawn("0.0.0.0:6663", &[], FailureDetectorConfig::default());
+        let node2 = ScuttleServer::spawn(
+            "0.0.0.0:6664",
+            &["0.0.0.0:6663".to_string()],
+            FailureDetectorConfig::default(),
+        );
+        let mut event_receiver = node1.scuttlebutt().lock().await.nodes_change_watcher();
+
+        // wait for nodes to exchange and try to get latest cluster change event.
+        tokio::time::sleep(Duration::from_secs(2)).await;
```

Comment:
Slow tests suck. There's a way to write this test so that we wait for the minimum amount of time required to observe the desired outcome. For instance:
```rust
        let mut live_nodes_watcher = node1
            .scuttlebutt()
            .lock()
            .await
            .nodes_change_watcher()
            .skip_while(|live_nodes| live_nodes.is_empty());

        tokio::time::timeout(Duration::from_secs(5), async move {
            let live_nodes = live_nodes_watcher.next().await.unwrap();
            assert_eq!(live_nodes.len(), 1);
            assert!(live_nodes.contains("0.0.0.0:6664"));
        })
        .await
        .unwrap();
```